modules/core/src/v2/coins/algo.ts

```diff
@@ -2,46 +2,44 @@
  * @prettier
  */
 import * as Bluebird from 'bluebird';
+import * as accountLib from '@bitgo/account-lib';
 import * as _ from 'lodash';
-import * as stellar from 'stellar-sdk';
-import { CoinFamily } from '@bitgo/statics';
-import {
-  NaclWrapper,
-  Multisig,
-  Address,
-  Seed,
-  generateAccountFromSeed,
-  generateAccount,
-  isValidAddress,
-  isValidSeed,
-  Encoding,
-  mergeMultisigTransactions,
-} from 'algosdk';
 import { BitGo } from '../../bitgo';
+import { SeedValidator } from '../internal/seedValidator';
+import { CoinFamily } from '@bitgo/statics';
 
 import {
   BaseCoin,
-  TransactionExplanation as BaseTransactionExplanation,
+  TransactionExplanation,
   KeyPair,
   ParseTransactionOptions,
   ParsedTransaction,
   VerifyAddressOptions,
   VerifyTransactionOptions,
   SignedTransaction,
+  TransactionRecipient,
   SignTransactionOptions as BaseSignTransactionOptions,
 } from '../baseCoin';
 import { KeyIndices } from '../keychains';
 import { NodeCallback } from '../types';
-import { SeedValidator } from '../internal/seedValidator';
 
 const co = Bluebird.coroutine;
 
-export interface TransactionExplanation extends BaseTransactionExplanation {
-  memo: string;
+export interface AlgoTransactionExplanation extends TransactionExplanation {
+  memo?: string;
   type?: string;
-  senderAddress?: string;
-  voteKeyBase64?: string;
-  voteLastBlock?: number;
+  voteKey?: string;
+  selectionKey?: string;
+  voteFirst?: number;
+  voteLast?: number;
+  voteKeyDilution?: number;
+  tokenId?: number;
+  operations?: TransactionOperation[];
+}
+
+export interface TransactionOperation {
+  type: string;
```

Comment:
Good opportunity here to use the `TransferType` type I suggested in the following comment.